src/components/pill/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef, useCallback } from "react"
+import { TextMicro } from "src/components/typography"
+import { Icon } from "src/components/icon"
+import Container from "./container"
+
+const Pill = forwardRef(({ children, background, color, hollow, icon, reverse, ...rest }, ref) => {
+  const { flavour } = rest
+
+  const renderIcon = useCallback(() => {
```

Comment:
Isn't it better to put it out of the Pill as its own component?

Reply:
Yeap, I totally agree 💯